sys-fs/lvm2/lvm2-2.03.16-r1.ebuild

```diff
@@ -219,34 +221,40 @@ src_install() {
 	local INSTALL_TARGETS=( install install_tmpfiles_configuration )
 	# install systemd related files only when requested, bug #522430
 	use systemd && INSTALL_TARGETS+=( SYSTEMD_GENERATOR_DIR="$(systemd_get_systemgeneratordir)" install_systemd_units install_systemd_generators )
```

Comment:
I wonder if it'd (all these INSTALL_TARGET appends) look better in a big array with usev/echo but let's not worry for now

Reply:
dunn